src/firefox/ffBrowser.ts

```diff
@@ -45,6 +45,8 @@ export class FFBrowser extends BrowserBase {
       promises.push((browser._defaultContext as FFBrowserContext)._initialize());
     }
     await Promise.all(promises);
+    if (browser._defaultContext && options.verifyPersistentContext)
```

Comment:
When does it exist and does not require "verification"?

Reply:
Undid this part, now it is clear.